chips/sam4l/src/pm.rs

```diff
@@ -475,6 +476,9 @@ unsafe fn configure_external_oscillator_pll() {
 pub unsafe fn setup_system_clock(clock_source: SystemClockSource, frequency: u32) {
     SYSTEM_FREQUENCY.set(frequency);
 
+    // For now, always go to PS2 as it enables all core speeds
```

Comment:
Seems reasonable as it's a strict improvement, and we clearly don't know exactly how this all works. Eventually we'll probably want to do something smarter based on the frequency requested, but not right now.